mspray/apps/main/tests/test_utils.py

```diff
@@ -383,3 +385,23 @@ def test_get_spray_operator(self):
         self.assertEqual(get_spray_operator("01234").pk, operator.pk)
         self.assertEqual(get_spray_operator("1234").pk, operator.pk)
         self.assertIsNone(get_spray_operator("01234X"))
+
+    def test_spray_operator_performance_report(self):
+        """
+        Test that performance_report actually updates an existing record.
```

Comment:
This test does not achieve the goal of "Test that performance_report actually updates an existing record."